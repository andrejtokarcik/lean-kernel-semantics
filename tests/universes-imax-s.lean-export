1 #NS 0 l1
2 #NS 0 l2
1 #US 0
2 #US 1
3 #UP 1
4 #UP 2
5 #UM 2 3
6 #UIM 5 2
