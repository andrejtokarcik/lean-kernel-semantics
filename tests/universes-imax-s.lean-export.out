<T>
  <k>
    .
  </k>
  <names>
    0 |-> *empty*
    1 |-> l1
    2 |-> l2
  </names>
  <universes>
    0 |-> 0
    1 |-> 1
    2 |-> 2
    3 |-> l1
    4 |-> l2
    5 |-> Max ( 2 , l1 )
    6 |-> Max ( Max ( 2 , l1 ) , 2 )
  </universes>
  <expressions>
    .Map
  </expressions>
  <def-types>
    .Map
  </def-types>
  <def-values>
    .Map
  </def-values>
  <exit-code>
    0
  </exit-code>
</T>
